docs/MainSite/Main.md

```diff
@@ -87,25 +91,30 @@ Navigating the Documentation {#navigate_documentation_sec}
 -----
 
 The SpECTRE documentation is organized into tutorials, developer guides, groups
-of related code, namespaces, and by files for easier navigation. For someone
-looking to run simulations with SpECTRE we recommend starting with the
-\ref tutorials "tutorials", which are located under the `Documentation` menu
-item at the top. The tutorials are designed to get users up and running with a
-simulation, and analyzing and visualizing the output. For people looking to
-contribute to SpECTRE there are tutorials under the \ref dev_guide "Dev Guide"
-part of the `Documentation` menu. The dev guide includes things such as
-\ref code_review_guide "code review guidelines", how to \ref writing_unit_tests
-"write unit tests", how to \ref writing_good_dox "write documentation", as well
-as some information about C++ and the philosophy behind SpECTRE development. The
-[Reference](modules.html) sections contains groups (managed through doxygen
-groups) of related code. For example, there is a
-group for all the data structures we use, a group for utility functions and
-classes, as well as groups for coordinate maps, domain creation, and many
-others. The [Reference](modules.html) is designed to help developers discover
-existing functionality so that things are not re-implemented several times. You
-can also get an overview of the code base by namespace by visiting the
-[Namespaces](namespaces.html) section under the `Reference` menu item. Finally,
-it is also possible to browse the repository by files under the
-[Files](files.html) menu item, though it is recommended that in that case you
-browse the [GitHub repository](https://github.com/sxs-collaboration/spectre)
-directly.
+of related code, namespaces, and files for easier navigation.
```

Comment:
... navigation.  These can all be accessed by links in the menu bar.  Below we describe what each menu item provides.